code/game/gamemodes/clock_cult/clock_structure.dm

```diff
@@ -29,7 +29,8 @@
 	return ..()
 
 /obj/structure/destructible/clockwork/ratvar_act()
-	obj_integrity = max_integrity
+	if(ratvar_awakens || clockwork_gateway_activated)
+		obj_integrity = max_integrity
```

Comment:
If you want it to be indestructible when the rat is awake, you should probably block the base calls of obj_break and obj_destruction if he is.